src/main/java/uk/ac/ebi/atlas/experimentpage/markergenes/MarkerGenesDao.java

```diff
@@ -17,11 +17,19 @@ public MarkerGenesDao(NamedParameterJdbcTemplate namedParameterJdbcTemplate) {
     }
 
     private static final String SELECT_MARKER_GENES_WITH_AVERAGES_PER_CLUSTER =
-            "SELECT * " +
-                    "FROM scxa_marker_gene_stats " +
-                    "WHERE k_where_marker = :k and experiment_accession = :experiment_accession AND marker_p_value < 0.05";
+            "SELECT g.experiment_accession, m.gene_id, " +
+                    "g.variable as k_where_marker, h.value as cluster_id_where_marker, " +
+                    "g.value as cluster_id, m.marker_probability as marker_p_value, " +
+                    "s.mean_expression, s.median_expression " +
+                    "FROM " +
+                    "scxa_cell_group_marker_gene_stats s, scxa_cell_group_marker_genes m, " +
+                    "scxa_cell_group g, scxa_cell_group h " +
```

Comment:
`scxa_cell_group h` and `scxa_cell_group g` are identical.

Reply:
True, this one suggested by Jon in this document: [SCXA Schema](https://docs.google.com/document/d/1hI8la-cxsElWgOErU1XTcc5KBuFY5O8_7aWcysQ9VfI/edit#heading=h.c38cam9l3xwi)

Reply:
He has provided a `Note` for that.

Reply:
The note put ```The marker stats table references the cell group table twice, once directly, once indirectly via the marker table (which records the precise grouping in which a gene was a marker).```

@alfonsomunozpomer how reference the cell group table indirectly?